app/controlplane/extensions/core/dependency-track/v1/README.md

```diff
@@ -0,0 +1,84 @@
+### Dependency-Track fan-out Extension
+
+This extension implements sending cycloneDX Software Bill of Materials (SBOM) to Dependency-Track. 
+
+See https://docs.chainloop.dev/guides/dependency-track/
+
+
```

Comment:
Should we have a separate section with an info on which materials, attestations are supported? 

```
### Supported metadata
The following metadata are supported by this extension:

Materials: CycloneDX
Attestations:  unsupported
```

Reply:
Yeah, this could be used as a baseline. The scope was just schema for now. but I could see value in generating `description` input types and so on. Should we do it in another diff?